CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ All changes should be developed on a feature branch, in order to keep them separ
 To create a new feature branch:
 
 ```bash
-git checkout -b my-new-feature upstream/master
+git checkout -b my-new-feature gwastro/master
```

Comment:
I still think a `git fetch gwastro` should be added just before this. I'm just worried newbies might think that checkout will automatically do the fetch. Also, if they're reading the help, they might skip right to the "Creating a new feature branch" section without seeing the "Updating your fork" section above it. If you add it, worse case scenario is someone runs fetch twice, but there's no harm in that.